src/main/webapp/js/userMapData.json

```diff
@@ -1110,6 +1165,7 @@
     "University of Colorado",
     "University of Colorado at Boulder",
     "University of Colorado at Colorado Springs",
+    "University of Colorado-Leeds School of Business",
```

Comment:
I checked that this is a campus of `University of Colorado at Boulder`, so it should not be duplicated.